plugins/core/IO/qE57IO/src/E57Filter.cpp

```diff
@@ -1399,6 +1399,10 @@ static bool GetPoseInformation(const e57::StructureNode& node, ccGLMatrixd& pose
 	bool validPoseMat = false;
 	if (node.isDefined("pose"))
 	{
+		CCCoreLib::SquareMatrixd rotMat(3);
```

Comment:
Is it necessary do that here since 'validPoseMat' is false? You could simply call 'validPoseMat.toIdentity()' at the beginning of the method, just in case.

Reply:
I was considering the case that translation is defined, but not rotation.  Seems reasonable to initialise to identity matrix, in the absence of rotation?

Reply:
Yes it's enough to initialize poseMat to identity directly.

Reply:
Updated as discussed.